backend/accounts_test.go

```diff
@@ -1506,3 +1507,17 @@ func TestAccountsTotalBalanceByKeystore(t *testing.T) {
 	require.NotNil(t, totalBalance[hex.EncodeToString(ks2Fingerprint)])
 	require.Equal(t, "0.13", totalBalance[hex.EncodeToString(ks2Fingerprint)].Total)
 }
+
+func TestConvertAmountToFiat(t *testing.T) {
+	b := newBackend(t, testnetDisabled, regtestDisabled)
+	defer b.Close()
+	b.ratesUpdater = rates.MockRateUpdater()
+	defer b.ratesUpdater.Stop()
+	// convert 1 BTC to USD
+	amount := coinpkg.NewAmountFromInt64(100000000)
+	converted, err := b.convertAmountToFiat(amount, "USD")
+	expectedValue := big.NewRat(21, 1)
+
+	require.NoError(t, err)
```

Comment:
nit: this line should be right below b.convertAmountToFiat for better readability